OmsAgent/ImportGPGkey.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+if [ -z "$1" ]; then
+    echo "Usage:"
+    echo "   $0 PUBLIC_GPG_KEY"
+    exit 1
+fi
+
+if [ -z "$2" ]; then
+    KEYRING_NAME="keyring.gpg"
+else
+    KEYRING_NAME=$2
+fi
+
+TARGET_DIR="$(pwd -P)/keys"
+HOME=$TARGET_DIR gpg --no-default-keyring --keyring $TARGET_DIR/$KEYRING_NAME --import $1
```

Comment:
gpg will be present in all distros right?

Reply:
before enabling this, should be do one release where we check if gpg is present and add that to telemetry?

Reply:
yes, if it not present then the keyring.gpg will not created and the signing will fail.